core/timetrace.go

```diff
@@ -189,6 +211,12 @@ func (report *Report) FormatCurrentTime() string {
 	return formatDuration(*report.TrackedTimeCurrent)
 }
 
+// FormatBreakTime returns the formated string of the total time
+// taking breaks today following the format convention
+func (report *Report) FormatBreakTime() string {
+	return formatDuration(report.BreakTimeToday)
+}
+
```

Comment:
Those formatting-related functions should live in `formatter.go` from now on. There's also #101 to move those existing functions.

Reply:
I moved the Format* functions into `formatter.go` in commit 3536a41. They are now methods of Formatter and take a `*Report` as an arg.